### ISSUE TITLE
Updated manuscript: CCDL Authors CRediT Taxonomy

### ISSUE THREAD

**Issue by jaclyn-taroni** [AlexsLemonade/OpenPBTA-manuscript#110]
We are planning to use the [CRediT taxonomy](https://www.cell.com/pb/assets/raw/shared/guidelines/CRediT-taxonomy.pdf) for author contributions.

| Role                       | Authors |
|----------------------------|---------|
| Conceptualization          |       |
| Data curation              |      |
| Formal Analysis            |      |
| Funding acquisition        |        |
| Investigation              |       |
| Methodology                |        |
| Project administration     |     |
| Resources                  |      |
| Software                   |         |
| Supervision                |    |
| Validation                 |      |
| Visualization              |         |
| Writing – original draft   |         |
| Writing – review & editing |         |

See the PDF linked above for definitions.

**Comment by jaclyn-taroni** [comment]
Jaclyn Taroni contributor roles:

- Methodology - substantive scientific review of multiple modules
- Software - continuous integration infrastructure, Docker
- Validation - if review for correctness of implementation and reproducibility of results qualifies as validation
- Formal analysis - focal CN preparation, molecular subtyping, transcriptomic dimension reduction, GISTIC
- Data curation - HGG subtyping, embryonal subtyping, compiling subtyping results
- Writing - Review and editing
- Visualization - transcriptomic overview figure included in current draft
- Supervision - oversee the CCDL team sprints
- Project administration - Coordination of analytical code review

ORCID (for my own convenience later): `0000-0003-4734-4508`

I am a bit fuzzy on our use of `Investigation` - which seems to be mostly related to QC and benchmarking (#107) - but more generally I would expect substantial overlap with `Formal Analysis`

**Comment by jashapiro** [comment]
Joshua A. Shapiro (include initial pls.) 

ORCID: `0000-0002-6224-0347`

Roles (tentative... willing to give some up if ambiguous):
- Software - Docker, sqlite, misc code 
- Validation - review and sanity checks...
- Formal analysis - interaction analysis, CNV consensus
- Investigation - CNV consensus, SNV consensus
- Writing - Original Draft (if methods sections count, which I assume it does)
- Writing - Review and editing
- Visualization - interaction plots
- Supervision - mentorship in CNV analysis, specifically?

Some of those may be questionable, depending on definitions, but I think that is a start. I definitely think that any time you look at the data and build analysis based on that, that has to count toward Investigation, don't you?

**Comment by cansavvy** [comment]
Candace L. Savonen

ORCID: `0000-0001-6331-7070`

I have not kept track of what changes have occurred with mutational signatures so perhaps the original analysis I did is invalid and shouldn't be credited? Not sure but I listed it for now. Never done this before so unsure if this is correct. 

Tentative roles:
- Methodology - SNV consensus, TMB calculations and comparison, Mutational signatures, chromosomal instability, survival analysis (framework/foundation only, not full implementation)
- Software - Misc code for the analyses listed above, and the figure generation script, 
- Validation - Code review for some snv callers, subtyping, plotting, and other miscellaneous code
- Formal analysis - SNV consensus, TMB calculations and comparison, Mutational signatures, chromosomal instability
- Investigation - SNV consensus, TMB calculations and comparison, Mutational signatures, chromosomal instability
- Writing - Original Draft
- Visualization -  TMB calculations and comparison, mutational signatures, chromosomal instability, CN status heatmap

**Comment by cbethell** [comment]
Chante J. Bethell

ORCID: `0000-0001-9653-8128`

- Methodology - focal CN preparation, molecular subtyping (ATRT and HGG), sample distribution analysis, transcriptomic dimension reduction
- Software - Miscellaneous code for the above analyses
- Validation - Code review for some plotting, subtyping and other miscellaneous code like sanity checks
- Formal analysis - focal CN preparation, molecular subtyping (ATRT and HGG), sample distribution analysis, transcriptomic dimension reduction
- Writing - Original Draft (Focal Copy Number Calling subsection of methods)
- Visualization - OpenPBTA distribution and OncoPrint figures included in current draft

Not completely sure if I accurately filled out the above  (considering I filled `Methodology` and `Formal analysis` out the same) so feel free to correct me if I am wrong!

**Comment by jharenza** [comment]
> I definitely think that any time you look at the data and build analysis based on that, that has to count toward Investigation, don't you?

@jaclyn-taroni yes, I agree with the above. The definition has its own term inside of it 😆 :

```
Conducting a research and investigation process, specifically performing the
experiments, or data/evidence collection 
```

I took it to mean that deeper "inspection" of the unexpected data/analyses/pathology would fall under this... we could probably clarify that term, since it is a bit more vague for computational work?

**Comment by jaclyn-taroni** [comment]
Looking at the definition of `Software`:

> Programming, software development; designing computer programs; implementation of the computer code and supporting algorithms; testing of existing code components

in combination with how this being used on #107 – given that this is an analysis paper, I take this to mean that `Software` covers items related more to infrastructure (additional packages, Docker, CI) but not the analytical code having to with the application of computational techniques which is covered by `Formal analysis`. 

@jharenza did I follow how D3b/DBHI's use of `Software` correctly?

**Comment by jaclyn-taroni** [comment]
I am going to move forward with a PR for these authors that interprets investigation more broadly ("any time you look at the data and build analysis based on that") and software narrowly (e.g., infrastructure) but I have filed #116 and #117 to track clarifying those roles.

**Comment by jaclyn-taroni** [comment]
Closed via #118 - changes might need to be made once #116 and #117 are resolved, though!
